atomic_reactor/plugins/build_source_container.py

```diff
@@ -43,10 +41,6 @@ def export_image(self, image_output_dir: Path):
             self.log.error("failed to save docker-archive :\n%s", e.output)
             raise
 
-        img_metadata = get_exported_image_metadata(str(output_path), IMAGE_TYPE_DOCKER_ARCHIVE)
-        # OSBS2 TBD exported_image_sequence will not work for multiple platform
-        self.workflow.data.exported_image_sequence.append(img_metadata)
-
```

Comment:
What do you think to save the metadata as part of the plugin result? This would be easy for koji_import to collect source image metadata. I'm thinking this because, for binary build, `fetch_docker_archive` plugin generates each downloaded image archive's metadata and returns the metadatas as plugin result.

We could keep this behavior consistently. That is, before collecting an image build's metadata, the built image's metadata is generated in advance in some plugin and is accessible at any time in subsequent plugins' execution.

Reply:
updated to return metadata.